.github/workflows/valgrind.yml

```diff
@@ -0,0 +1,116 @@
+name: Valgrind
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: PHP ${{ matrix.php-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - '5.6.40'
+          - '7.4.19'
+          - '8.0.7'
```

Comment:
Very specific versions?

Reply:
They have to be unless there is a URL that gets the latest PHP 8.0 release etc.

Reply:
Couldn't find an alternative.